### PR TITLE
Splitting page content

### PR DESCRIPTION
**Pull request by gingerzoealex**
A new page for meetups. Gives us room to add more details without having too much content per page. The specific groups might need updated though.

### CODE REVIEW

File: events/how-to-start-a-class.md [hunk 1]
```diff
@@ -43,24 +43,3 @@ If you are are interested in running a course on our premises then contact [even
 
 # Want to start something?
```
Comment (andrewbolster) [inline events/how-to-start-a-class.md]: With the deletion, there's no content after this header, which isn't good...

Comment (gingerzoealex) [reply events/how-to-start-a-class.md]: I'll fix it and commit again, sorry!

Comment (gingerzoealex) [reply events/how-to-start-a-class.md]: Done, let me know if it's ok

Comment (gingerzoealex) [reply events/how-to-start-a-class.md]: It's showing in the log that it's been committed to this branch but it's not listed in commit. Not sure what to do now :p

Comment (gingerzoealex) [reply events/how-to-start-a-class.md]: Nevermind, got it!

File: events/meetups.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+title: Meetup Groups
```
Comment (andrewbolster) [inline events/meetups.md]: Is "Meetup" not a bit brand specific? Might exclude people who aren't using Meetup. What about "Meetups and User Groups"?

Comment (gingerzoealex) [reply events/meetups.md]: I thought meetup was the general term for "user group"?

Comment (gingerzoealex) [reply events/meetups.md]: But I see what you mean anyway.. I'll commit again and change it to "Meetups and User Groups". Any other changes before I do it?

**Pull request closed without merging**